CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix: Dual core devices had incorrect 'core' names in `STM32H7_Series.yaml`, causing panic during flashing. (#1023)
 - Fix: Include all RAM regions in `STM32H7_Series.yaml` (#429)
 - Fix: Include all new STM32H7 variants from the latest CMSIS pack file (#913)
+- Fix: Update STM32G0_Series.yaml to include latest variants (STM32G050, STM32G051, STM32G061, STM32G0B0, STM32G0B1, STM32G0C1)
```

Comment:
Minor request ... can you please add the PR number at the end of the CHANGELOG line, as in other lines.

Thanks for your patience and contribution.

Reply:
> Thanks for your patience and contribution.

no problem, thanks to whoever made this change easy enough for me to do, and to y'all for guiding me through the process :)